github/resource_github_branch_protection_v3_utils.go

```diff
@@ -191,8 +209,56 @@ func expandRequiredStatusChecks(d *schema.ResourceData) (*github.RequiredStatusC
 			m := v.(map[string]interface{})
 			rsc.Strict = m["strict"].(bool)
 
+			// Initialise empty literal to ensure an empty array is passed mitigating schema errors like so:
+			// For 'anyOf/1', {"strict"=>true, "checks"=>nil} is not a null. []
+			rscChecks := []*github.RequiredStatusCheck{}
+
+			// TODO: Remove once contexts is deprecated
+			// Iterate and parse contexts into checks using -1 as default to allow checks from all apps.
```

Comment:
Is this -1 value documented on the GitHub API? Do you know where it comes from?

Reply:
Yes it is :-) 
https://docs.github.com/en/rest/branches/branch-protection?apiVersion=2022-11-28#update-branch-protection
![image](https://user-images.githubusercontent.com/30706552/211641998-498ff75b-a934-4f17-9967-b394213b127d.png)